CHANGELOG.md

```diff
@@ -8,7 +8,7 @@ _**Note:** Yet to be released changes appear here._
 
 ## 11.2.1
 
-* `FIX`: correct popup menu icon to label spacing
+* `FIX`: correct popup menu icon to label spacing ([da5dac5](https://github.com/bpmn-io/diagram-js/commit/da5dac5))
```

Comment:
```suggestion
* `FIX`: correct popup menu icon to label spacing ([`da5dac5`](https://github.com/bpmn-io/diagram-js/commit/da5dac5))
```

Reply:
I fixed this via https://github.com/bpmn-io/diagram-js/pull/721/commits/7dad62e31051e39f65f401cf32df1375831a974e